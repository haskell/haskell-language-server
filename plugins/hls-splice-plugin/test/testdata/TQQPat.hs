{-# LANGUAGE QuasiQuotes #-}
module TQQPat where
import QQ

f :: String -> IO ()
f [str|str|] = putStrLn "is str"
f _ = putStrLn " not str"
