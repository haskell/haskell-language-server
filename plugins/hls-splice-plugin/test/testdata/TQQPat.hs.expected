{-# LANGUAGE QuasiQuotes #-}
module TQQPat where
import QQ

f :: String -> IO ()
f "str" = putStrLn "is str"
f _ = putStrLn " not str"
